src/Illuminate/Database/Schema/MySqlBuilder.php

```diff
@@ -31,18 +31,31 @@ public function dropDatabaseIfExists($name)
     }
 
     /**
-     * Determine if the given table exists.
+     * Get the tables for the database.
      *
-     * @param  string  $table
-     * @return bool
+     * @return array
      */
-    public function hasTable($table)
+    public function getTables()
```

Comment:
Breaking change (for 10.x), right?

Reply:
Ofcourse not, `MySqlByilder` extends `Builder` class, there is no need to overide `hasTable` on `MySqlBuilder` anymore, as it would be the same as implemented on the parent class. Also we already have tests for this.